tests/unit/test_sqlite_graph.py

```
@@ -441,6 +441,31 @@ async def test_list_edges_for_memories_chunking_boundaries(
         res_901 = await adapter.list_edges_for_memories(ids_901)
         assert len(res_901) == 900
 
+    async def test_list_edges_for_memories_cross_chunk(self, adapter: SQLiteGraphAdapter) -> None:
+        """チャンクを跨ぐエッジが取得されることを検証する."""
+        # 1000個のノードを作成 (CHUNK_SIZE=900 を想定)
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `ノ` (KATAKANA LETTER NO). Did you mean `/` (SOLIDUS)?

(RUF003)